components/Operations/tezConnections.ts

```diff
@@ -113,11 +129,29 @@ export function updateStakingOpsStatus(
       return operation
     })
   }
-  // if prevBakerAddress is not null then check if the current baker address matches the prevBakerAddress then only CanStake should be true.
-  if (prevBakerAddress) {
-    opStatus.CanStake =
-      opStatus.CanStake && prevBakerAddress === delegate.address
+
+  if (type === 'user') {
+    opStatus.pendingUnstakeOpsWithAnotherBaker =
+      (prevBakerAddress &&
+        (!Boolean(delegate) || prevBakerAddress !== delegate?.address)) ??
+      false
+    opStatus.bakerAcceptsStaking =
+      (Boolean(delegate) &&
```

Comment:
!!delegate && !!bakerList?.find(baker => baker.address === delegate.address)?.acceptsStaking)

Reply:
Done except used Boolean in place of !!